src/jdk.jshell/share/classes/jdk/jshell/execution/LocalExecutionControl.java

```diff
@@ -90,11 +94,19 @@ public void load(ClassBytecodes[] cbcs)
     private static byte[] instrument(byte[] classFile) {
         var cc = ClassFile.of();
         return cc.transformClass(cc.parse(classFile),
-                        ClassTransform.transformingMethodBodies((cob, coe) -> {
-                            if (coe instanceof BranchInstruction)
-                                cob.invokestatic(CD_Cancel, "stopCheck", ConstantDescs.MTD_void);
-                            cob.with(coe);
-                        }));
+                        ClassTransform.transformingMethodBodies(
+                            CodeTransform.ofStateful(() -> {
+                                HashSet<Label> priorLabels = new HashSet<>();
```

Comment:
Nit: could be `Set<Label> priorLabels = new HashSet<>();`.

Reply:
Thanks - fixed in 16a39b28197.